app/views/templates/payments/whatYouOweChargeRow.scala.html

```diff
@@ -24,42 +24,42 @@
 @chargeDetails = @{new WhatYouOweChargeHelper(payment, messages)}
 @defining(chargeDetails.description()) { chargeDescription =>
 
-<tr id="payment-@{count + 1}">
-    <td scope="row">
+<div id="payment-@{count + 1}">
 
-        <h2 class="bold heading-small">@chargeDetails.title</h2>
-        @chargeDescription.map { description => <div>@description</div> }
-
-        <div data-due="@(payment.due.toString)" class="form-hint">
-            @messages("openPayments.dueBy") @displayDate(payment.due,useShortDayFormat = true)
-            @if(payment.overdue) {
+<dl class="govuk-check-your-answers cya-questions-long what-you-owe-dl">
+    <div class="check-your-answer-row what-you-owe-row">
+        <dt class="cya-question what-you-owe-due-date">
+            <h2 class="bold heading-small what-you-owe-heading">@chargeDetails.title @chargeDescription.map { description => <div class="what-you-owe-subheading">@description</div> }</h2>
+            <div data-due="@(payment.due.toString)" class="form-hint">
+                @messages("openPayments.dueBy") @displayDate(payment.due,useShortDayFormat = true)
+                @if(payment.overdue) {
                 <span class="inline"><strong class="task-overdue">@messages("common.overdue")</strong></span>
-            }
-        </div>
-    </td>
-
-    <td scope="row" class="numeric nowrap">
-        <span data-amount="@(payment.amount)">@displayMoney(payment.amount)</span>
-    </td>
-
-    <td scope="row" class="nowrap">
-        <div>
-            <a data-journey-click="payment:make-payment:open-payments" href="@payment.makePaymentRedirect">
-                <span>@chargeDetails.payLinkText</span>
-                <span class="visuallyhidden">
+                }
+            </div>
+        </dt>
+        <dd class="cya-answer what-you-owe-amount">
+            <span data-amount="@(payment.amount)">@displayMoney(payment.amount)</span>
+        </dd>
+        <dd class="cya-change what-you-owe-links">
+            <div>
+                <a data-journey-click="payment:make-payment:open-payments" href="@payment.makePaymentRedirect">
+                    <span>@chargeDetails.payLinkText</span>
+                    <span class="visuallyhidden">
```

Comment:
One of the issues DAC raised is that the visually hidden screen reader content for the "Pay now" link was too verbose and that we only needed the amount. Could you remove the two unnecessary lines from this `visuallyhidden` span please.

Could you also remove the val `overdueContext` from `WhatYouOweChargeHelper`.